storagenode/reputation/reputation.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package reputation
+
+import (
+	"context"
+	"time"
+
+	"storj.io/storj/pkg/storj"
+)
+
+// DB works with reputation database
+type DB interface {
+	// Store inserts or updates reputation stats into the DB
+	Store(ctx context.Context, stats Stats) error
```

Comment:
Currently there's a mismatch in naming. Usually there are names paired together `Store`/`Load` and `Set`/`Get` or `Put`/`Get`.

In this case probably `Set` makes more sense.